subsys/net/lib/nrf_provisioning/src/nrf_provisioning.c

```diff
@@ -512,6 +512,12 @@ static void commit_latest_cmd_id(void)
 	}
 }
 
+void nrf_provisioning_set_interval(int interval)
+{
+	nxt_provisioning = interval;
+	LOG_DBG("Provisioning interval set to %d", interval);
```

Comment:
Don't we need to re-schedule here?

Reply:
Updated the function to reschedule and now the interval is saved to storage.